src/main/java/core/basesyntax/service/impl/FruitTransactionServiceImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionService;
+
+public class FruitTransactionServiceImpl implements FruitTransactionService {
+    @Override
+    public FruitTransaction createTransaction(String operationCode, String fruit, int quantity) {
```

Comment:
This service and method is redundant. It contains a simple constructor. You can move code from this method to the ParserServiceImpl.pareseData() method